docs/api/helpers/DirectionalLightHelper.html

```diff
@@ -32,13 +32,13 @@
 		<h2>Constructor</h2>
 
 
-		<h3>[name]( [page:DirectionalLight light], [page:Number size], [page:Hex overrideColor] )</h3>
+		<h3>[name]( [page:DirectionalLight light], [page:Number size], [page:Hex color] )</h3>
 		<div>
 			[page:DirectionalLight light]-- The light to be visualized. <br /><br />
 
 			[page:Number size] -- (optional) dimensions of the plane. Default is *1*.<br /><br />
 			
-			[page:Hex overrideColor] -- (optional) by default the helper will take the color of the light.
+			[page:Hex color] -- (optional) by default the helper will take the color of the light.
 				In cases where this makes the helper difficult to see 
 				(for example a white light against a white background) you can directly set the color here. 
```

Comment:
Just say: (optional), otherwise the helper takes the color of the light, and leave it at that.